apps/nextjs/src/middlewares/csp.ts

```diff
@@ -1,56 +1,70 @@
 import { NextRequest, NextResponse } from "next/server";
 
-const sentryEnv = process.env.NEXT_PUBLIC_SENTRY_ENV;
-const sentryRelease = process.env.NEXT_PUBLIC_APP_VERSION;
-const sentryReportUri = `${process.env.SENTRY_REPORT_URI}&sentry_environment=${sentryEnv}&sentry_release=${sentryRelease}`;
+function generateNonce(): string {
```

Comment:
I'm not too happy about this. Because it's an edge function, and we're running the test in node we need a way to generate the nonce in both environments.